code/obj/item/pens_writing_etc.dm

```diff
@@ -41,6 +41,7 @@
 	var/spam_timer = 20
 	var/symbol_setting = null
 	var/material_uses = 10
+	var/fits_on_things = TRUE // for a whitelist of pens that can fit in PDAs and clipboards
```

Comment:
Nitpick 1: So which is it... Does it fit in? Or fit on?
Nitpick 2: Calling it a whitelist makes me think a list is going to be checked somewhere.

Reply:
don't gotta worry about this now that it dead :))